mx.py

```diff
@@ -3854,11 +3856,12 @@ def get_os():
 def get_os_variant():
     global _os_variant
     if _os_variant is None:
-        _os_variant = get_os()
-        if _os_variant == 'linux' and 'musl' in _check_output_str(['ldd', sys.executable]):
-            _os_variant = 'linux-musl'
+        if get_os() == 'linux' and 'musl' in _check_output_str(['ldd', sys.executable]):
+            _os_variant = 'musl'
+        else:
+            _os_variant = ''
         if _opts and _opts.verbose:
```

Comment:
Any reason why you don't use `logv` instead of checking for `_opts.verbose`?

Reply:
No, just reused this pattern from somewhere I saw it